packages/overlay/src/Overlay.ts

```diff
@@ -536,6 +537,9 @@ export class Overlay extends OverlayFeatures {
         if (!this.open) {
             document.body.offsetHeight;
         }
+        /**
```

Comment:
Why can't we move this custom Event to surface to the react events?

Reply:
There’s not currently tooling to surface enough data to do so. We need to spend some time seeing if we can support brining that to life, but I wanted to make sure the library was in a publishable form while we investigate that. For the last two releases I had corrected it manually, but I can never be sure that it really worked as we don’t have any (post publication) testing for the wrappers.